tools/parse/src/main/scala/org/allenai/nlpstack/parse/PolytreeParser.scala

```diff
@@ -39,39 +39,45 @@ class PolytreeParser(
       polyparser.Parser.loadParser(parserConfig)
     } else {
       val cachedParses = MultiPolytreeParseSource(cacheFiles map { filename =>
-        FileBasedPolytreeParseSource(filename.toString, ConllX(true))
+        FileBasedPolytreeParseSource(filename.toString, ConllX(true, makePoly = true))
```

Comment:
It reads the ConllX as a polytree-structured parse, rather than a singly-rooted parse. The PolytreeParser API turns this back into a singly-rooted parse before returning.